### DISCUSSION TITLE
Where is the Last Desert Ultra-Marathon held? 🏜️🏃‍♂️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1260] [Galaxy Trivia] [answered]
**Bonus:** Which country claims more than 60% of the world's lakes?

**Comment by tomerf** [comment]
The last desert ultra marathon was held in Namibia.

Bonus: of the more than 1.4 million lakes in the world that register more than 0.2 square miles, over 60% of them reside in Canada.

**Comment by peymansf2000** [comment, marked as answer]
The Last Desert Ultra-Marathon is held in Antarctica. It is part of the 4 Deserts series of ultramarathons.

As for the bonus question, Canada claims more than 60% of the world's lakes. Canada has numerous lakes, and it is estimated that around 60% of the world's lakes are located within its borders.

**Reply by ParanoidUser** [reply]
Bingo bango, you nailed it! 🎯
